zerver/lib/slack_data_to_zulip_data.py

```diff
@@ -498,9 +501,21 @@ def convert_slack_workspace_messages(slack_data_dir: str, users: List[ZerverFiel
 
     return message_json
 
+def get_all_messages(slack_data_dir: str, added_channels: AddedChannelsT) -> List[ZerverFieldsT]:
+    all_messages = []  # type: List[ZerverFieldsT]
+    for channel_name in added_channels.keys():
+        json_names = os.listdir(slack_data_dir + '/' + channel_name)
```

Comment:
We should use `os.path.join` here, since it's possible channels will have funny characters in them.  You can have more local variables, also, to reduce some repetition:

~~~ py
channel_dir = os.path.join(slack_data_dir, channel_name)
json_names = os.listdir(channel_dir)
for json_name in json_names:
    message_fn = os.path.join(channel_dir, json_name)